docker-compose.yml

```diff
@@ -83,6 +83,7 @@ services:
   '8.3-buster': { <<: *linux_php_service, image: 'datadog/dd-trace-ci:php-8.3_buster' }
   'php-master-buster': { <<: *linux_php_service, image: 'datadog/dd-trace-ci:php-master_buster' }
   # --- Bookworm ---
+  '8.2-bookworm': { <<: *linux_php_service, image: 'datadog/dd-trace-ci:php-8.2_bookworm-3' }
```

Comment:
Not really needed but I added it here because that's the image I use to test benchmarks. Appsec is not compiling on buster versions